File: discover_ruby/github_most_popular.rb
require 'httpclient'

extheaders = {
  'User-Agent' => 'Holberton_School',
  'Authorization' => 'token 576bcef0e6a9e47b7bbb47385326d42a630bfff6'
}

clnt = HTTPClient.new

puts clnt.get_content('https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc', query = nil, extheader = {})

File: discover_more_python/github_most_popular.py
request_headers = {
  'User-Agent': 'Holberton_School',
  'Authorization': 'token 42ee520bd99c8951da18894d2e0cd0c06d957089'
} 

import urllib2

def main():
  # open a connection to a URL using urllib2
  webUrl = urllib2.urlopen("https://api.github.com/search/repositories?q=language:python&sort=stars&order=desc")  

  # read the data from the URL and print it
  data = webUrl.read()
  print data

if __name__ == "__main__":
  main()
